lib/voxa-ga.js

```diff
@@ -73,6 +73,72 @@ function register(skill, config) {
     return send(event);
   });
 
+  if (Object.prototype.hasOwnProperty.call(skill, 'onAlexaSkillEvent.SkillEnabled')) {
```

Comment:
we tend to use `lodash` for this sort of thing, i'd say that we should use it here as well, for consistency

Reply:
I'll make the change